manifests/profile/quod/prod/perl.pp

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2020 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::quod::prod::perl
+#
+# Install perl dependencies for quod applications. These packages
+# and settings are unique to the prod  environments.
+#
+# @example
+#   include nebula::profile::quod::prod::perl
+class nebula::profile::quod::prod::perl () {
+  require nebula::profile::quod::dependencies::perl
+
+  package { [
+    'libarchive-extract-perl',
+    'libcgi-fast-perl',
+    'libcpan-meta-perl',
+    'libcrypt-ssleay-perl',
+    'libdata-section-perl',
+    'libfcgi-perl',
+    'liblog-message-perl',
+    'liblog-message-simple-perl',
+    'libmodule-build-perl',
+    'libmodule-signature-perl',
+    'libpackage-constants-perl',
+    'libpod-latex-perl',
+    'libpod-readme-perl',
+    'libregexp-common-perl',
+    'libsoftware-license-perl',
+    'libterm-ui-perl',
+    'libtest-tester-perl',
+    'libtext-soundex-perl',
+    'libtext-template-perl']:
+  }
+
+  -> nebula::cpan { [
```

Comment:
Same question as before; do we really need every single one of those packages installed before we can run `cpan -i` on these?

Reply:
I am basing this off of www_lib format to some degree.

Also, I am not sure which of the 200+ packages are prerequisites for the CPAN modules. 

Is there an issue if these are ordered? Cause find chain of dependencies can quite tedious and time consuming without much benefit normally.